shared/actions/fs/index.js

```diff
@@ -672,6 +672,17 @@ const letResetUserBackIn = ({payload: {id, username}}: FsGen.LetResetUserBackInP
 
 const letResetUserBackInResult = () => undefined // Saga.put(FsGen.createLoadResets())
 
+const deleteFile = (state, {payload: {path}}: FsGen.DeleteFilePayload) => new Promise(() => {
```

Comment:
Why an additional promise? Would something like this work?

```
const deleteFile = (state, {payload: {path}}: FsGen.DeleteFilePayload) => 
    RPCTypes.SimpleFSSimpleFSRemoveRpcPromise({
        opID: Constants.makeUUID(),
        ....
    })
```

Reply:
Sure, although given that I'll need to add a `Wait`, seems like it's going to need a bit of a refactor anyway.